src/OmnichannelChatSDK.ts

```diff
@@ -1477,7 +1481,7 @@ class OmnichannelChatSDK {
                 CustomerLocale: body.locale || getLocaleStringFromId(this.localeId)
             };
 
-            const emailResponse = this.OCClient.emailTranscript(
+            const emailResponse = await this.OCClient.emailTranscript(
```

Comment:
Why we are changing this

Reply:
It's a bug found in the code while working on this fix. 

Before, `this.OCClient.emailTranscript()` would be called and not wait until this call gives us a success or failure response before saying the email transcript scenario is completed. So, it was a false positive. 

To fix this, we need to wait until that call gets completed to return a success or failure response to return the right scenario marker (completed or failed)